### PR TITLE
Add scanning hints for apiserver-proxy envoy image.

### PR DESCRIPTION
**Pull request by axel7born**
**How to categorize this PR?**

/area networking
/kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

File: imagevector/images.yaml [hunk 1]
```diff
@@ -681,6 +681,28 @@ images:
     value:
     - type: 'githubTeam'
       teamname: 'gardener/gardener-core-networking-maintainers'
+  - name: 'cloud.gardener.cnudie/dso/scanning-hints/package-versions'
+    value:
+    # https://www.envoyproxy.io/docs/envoy/v1.29.2/intro/arch_overview/security/external_deps
```
Comment (ScheererJ) [inline imagevector/images.yaml:686]: Why does the version here not relate to the version in `tag` above`? Do you need to rebase?

Comment (axel7born) [reply imagevector/images.yaml:686]: Yes. Done.

**Review by ScheererJ — APPROVED** [review]
/lgtm
/approve

**Pull request merged**